Makefile

```diff
@@ -53,7 +53,7 @@ build-test: pod_test build archive init-sandbox
 	cd Examples/ArcSplitting && make all
 	cd Examples/React && make all
 	cd Examples/SwiftSubspec && make all
-	# cd Examples/FBSDK && make all
+	cd Examples/FBSDK && make all
```

Comment:
The CI is getting pretty slow, it might be better to hold off on adding this a build test until we can shard the CI jobs